processor/src/handler.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use sawtooth_sdk::messages::processor::TpProcessRequest;
+use sawtooth_sdk::processor::handler::ApplyError;
+use sawtooth_sdk::processor::handler::TransactionContext;
+use sawtooth_sdk::processor::handler::TransactionHandler;
+
+use bond_common::addressing::*;
+
+pub struct BondTransactionHandler {
+    family_name: String,
+    family_versions: Vec<String>,
+    namespaces: Vec<String>,
+}
+
+impl BondTransactionHandler {
+    pub fn new() -> BondTransactionHandler {
+        BondTransactionHandler {
+            family_name: "bond".to_string(),
+            family_versions: vec!["0.1".to_string()],
+            namespaces: vec![hash(&FAMILY_NAMESPACE, 6)],
```

Comment:
No to be to picky.. But I was wondering if there should be a get_family_namespace_hash method or something like that in addressing? Just so we don't need this hard coded 6 here. If we are abstracting away all the addressing stuff anyway...

Reply:
Yes, I initially had a prefix hash function that was basically the same as the hash function in the addressing file, which was consolidated into a single hash function after some review of that file. I can add back a prefix function which gets the bond namespace because I think, overall, that's a good approach. Especially if that namespace is being referenced in the processor.

Reply:
@dplumb94 thoughts?

Reply:
Lols yeas.. I remember I suggested you to remove it xD